tests/test_MocLogin.py

```diff
@@ -0,0 +1,101 @@
+from django.test import TestCase
+from core.models import AtmosphereUser as User
+from core.models import Provider
+from core.models import PlatformType
+from core.models import ProviderType
+
+from threepio import logger
+
+from atmosphere import settings
+
+
+# This is assuming a clean database
+class MocLogin(TestCase):
+    username = test_settings['username']
```

Comment:
you are still creating user in the backend. That user doesn't have any password stored and it is just holding the identity. And in identity, you are using that to link with credentials which has token stored, currently. And creating a dummy user is necessary and that DB should be deleted. It is a one-time db. just get rid of `--keepdb` in the run_test.sh